tests/.suitespec.json

```diff
@@ -648,6 +649,7 @@
             "@flask",
             "@appsec",
             "@wsgi",
+            "tests/appsec/*",
```

Comment:
maybe we can move this pattern to the `appsec` component to reduce duplication?

Reply:
For now, we are only supporting django and flask for threats. But this is definitely something we need to keep in mind when we extend asm support.